lib/python/src/bailo/core/utils.py

```diff
@@ -3,7 +3,7 @@
 from typing import Any
 import os
 
-NO_COLOR = int(os.environ.get("NO_COLOR", 0))
+NO_COLOR = bool(int(os.environ.get("NO_COLOR", 0)))
```

Comment:
This will Falsify strings such as "0". I would instead use something like, `"NO_COLOR" in os.environ`